compiler/src/main/scala/dxCompiler/Main.scala

```diff
@@ -902,8 +902,8 @@ object Main {
         |    -folder <string>         Platform folder (defaults to '/').
         |    -project <string>        Platform project (defaults to currently selected project).
         |    -language <string> [ver] Which language to use? May be WDL or CWL. You can optionally 
-        |                             specify a version. Currently, WDL draft-2, 1.0, and 1.1 are
-        |                             fully supported and WDL development and CWL 1.2 are partially
+        |                             specify a version. Currently: i. WDL: draft-2, 1.0, and 1.1, and
+        |                             ii. CWL: 1.2 are fully supported and WDL development is partially
```

Comment:
```suggestion
        |                             ii. CWL: 1.2 are supported and WDL development is partially
```
See comment here: https://github.com/dnanexus/dxCompiler/pull/319#discussion_r827459477